test/scripts/install-kind.sh

```diff
@@ -2,23 +2,30 @@
 
 set -ex
 
-export GO111MODULE="on"
+# Install latest stable kubectl
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
+chmod +x ./kubectl
+sudo mv ./kubectl /usr/local/bin/kubectl
 
-pushd $GOPATH/src/k8s.io/kubernetes/
-if [[ ! -f /usr/local/bin/kubectl ]]; then
-  sudo ln ./_output/local/go/bin/kubectl /usr/local/bin/kubectl
-fi
-if [[ ! -f /usr/local/bin/e2e.test ]]; then
-  sudo ln ./_output/local/go/bin/e2e.test /usr/local/bin/e2e.test
-fi
-popd
+# Install latest e2e test binary
+# The e2e binaries are built from https://github.com/aojea/kubernetes-e2e-binaries
+# Current e2e binary versions is latest 1.19 tag v1.19.4
+curl -LO https://github.com/aojea/kubernetes-e2e-binaries/releases/download/d360454c/e2e.test
```

Comment:
how big is this binary? could we just add it to this repo instead?

Reply:
130 mbs, but this is hosted in github and it takes only a few seconds to download
The nice thing here is that we can get the e2e test fixes or new tests from upstream just changing the url to the new binary, without having to compile the kubernetes source

Reply:
actually is 2 seconds to download it, you can click in the `kind-setup` step in one of the actions
```
 curl -LO https://github.com/aojea/kubernetes-e2e-binaries/releases/download/d360454c/e2e.test
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   634  100   634    0     0   4839      0 --:--:-- --:--:-- --:--:--  4839

 24  114M   24 27.7M    0     0  46.9M      0  0:00:02 --:--:--  0:00:02 46.9M
100  114M  100  114M    0     0  83.1M      0  0:00:01  0:00:01 --:--:--  110M
+ chmod +x ./e2e.test
+ sudo mv ./e2e.test /usr/local/bin/e2e.test
```